fe/fe-core/src/main/java/org/apache/doris/common/util/DynamicPartitionUtil.java

```diff
@@ -740,8 +740,7 @@ public static String getHistoryPartitionRangeString(DynamicPartitionProperty dyn
             date = simpleDateFormat.parse(time);
         } catch (ParseException e) {
             LOG.warn("Parse dynamic partition periods error. Error={}", e.getMessage());
-            return getFormattedTimeWithoutMinuteSecond(
-                    ZonedDateTime.parse(timestamp.toString(), dateTimeFormatter), format);
+            throw new AnalysisException("Parse dynamic partition periods error. Error=" + e.getMessage());
```

Comment:
Why changing this logic?

Reply:
because of timestamp is null at this time